composer.json

```diff
@@ -98,7 +98,7 @@
         "doctrine/reflection": "~1.0",
         "doctrine/doctrine-bundle": "~1.4",
         "monolog/monolog": "~1.11",
-        "ocramius/proxy-manager": "~0.4|~1.0|~2.0",
+        "ocramius/proxy-manager": "^2.1",
```

Comment:
This will only affect the development of Symfony as `require-dev` dependencies are not transitive. We also need to update the `conflict` rule.

Reply:
for a conflict rule the constraint should look like this:

```
"ocramius/proxy-manager": "<2.1"
```

Reply:
I'know. thanks for helping :) 

I've updated my pr